wrangler-core/src/test/java/io/cdap/wrangler/parser/MapArgumentsTest.java

```diff
@@ -110,4 +112,24 @@ public void testMultipleArgumentsOptional() throws Exception {
     Assert.assertEquals(true, arguments.contains("col4"));
   }
 
+  @Test
+  public void testTextLists() throws Exception {
+    Compiler compiler = new RecipeCompiler();
+    CompileStatus status = compiler.compile("remove-sensitive-data :column \"ALL_BASIC\", \"AGE\";");
+
+    UsageDefinition.Builder builder = UsageDefinition.builder("rename");
+    builder.define("column", TokenType.COLUMN_NAME);
+    builder.define("infoTypes", TokenType.TEXT_LIST);
+
+    Iterator<TokenGroup> iterator = status.getSymbols().iterator();
+    Arguments arguments = new MapArguments(builder.build(), iterator.next());
+    Assert.assertEquals(2, arguments.size());
+    Assert.assertTrue(arguments.contains("column"));
+    Assert.assertTrue(arguments.contains("infoTypes"));
+    List<String> infoTypes = ((TextList) arguments.value("infoTypes")).value();
```

Comment:
seems like the value for "column" should also be checked

Reply:
done!